phpBB/styles/prosilver/template/captcha_recaptcha.html

```diff
@@ -29,7 +29,7 @@ <h3 class="captcha-title">{L_CONFIRMATION}</h3>
 		</div>
 		</noscript>
 
-		<a href="http://www.google.com/intl/{LA_RECAPTCHA_LANG}/policies/" target="_blank" class="recaptcha-responsive" style="display: none"><img alt="" width="71" height="36" src="{RECAPTCHA_SERVER}/img/clean/logo.png"></a>
+		<a href="http://www.google.com/intl/{L_RECAPTCHA_LANG}/policies/" target="_blank" class="recaptcha-responsive" style="display: none"><img alt="" width="71" height="36" src="{RECAPTCHA_SERVER}/img/clean/logo.png"></a>
```

Comment:
this was intentionally, because it is inside a URL.
Since it should only contain a few characters to identify the language, it should be fine to still use `LA_`?